pegasus/sites.v3/code.org/views/stats_carousel.haml

```diff
@@ -9,12 +9,10 @@
         .slide
           .slide-img<
             %img{:src=>"/images/cs-stats/#{statistic[:code_s]}.png"}
-    %a.slide-prev{:href=>'#'}<
-      %span<
-        %span.label Previous
-    %a.slide-next{:href=>'#'}<
-      %span<
-        %span.label Next
+    %a.slide-prev{href:'#', aria:{label: 'View previous slide'}}
```

Comment:
LGTM!

 Non-blocking general question - do we need to think about translating aria labels?

Reply:
Since this page isn't currently translated (and is probably due for a redesign 😄) I didn't add translations this time, but I do translate aria labels on pages that are already translated or fully redesigned/refactored.